pkgs/development/python-modules/azure-mgmt-appconfiguration/azure-mgmt-apimanagement/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildPythonPackage, fetchPypi
```

Comment:
Duplicate file?

Reply:
oh yea, it is, messed up a cp command when I was fleshing out the dependencies

Reply:
removed